content/en/seachat/seachat-manual/03-add-knowledge/09-add-webpage-link-in-answers.md

```diff
@@ -92,5 +92,67 @@ Once there is information in the knowledge base, the agent will start using this
     <p style="margin-top: 20px; font-size: 15px">URLs added to Agent's Answer</p>
 </div>
 
+## URL Buttons from Spreadsheets
+
+You can also add URL buttons to your agent's responses using a spreadsheet. When you are adding knowledge using a spreadsheet to upload a large amount of information, it can be difficult to add reference buttons to each knowledge manually. Don't worry. SeaChat has a solution for you. 
+
+By adding a column called **seachat_ref** in your spreadsheet, your agent will automatically extract information to add URL buttons to the agent's responses. You can put as many column as you want to add multiple URL buttons to the answer.
+
+| seachat_ref      ||
+|------------------|-|
+| button text |
+|button value |
```

Comment:
can you make a real table or use a screenshot? the markdown format doesn't read like a table at all:

![image](https://github.com/seasalt-ai/wiki/assets/543547/4d6015de-e967-41f8-b9eb-cd0787ce8cb7)

Remember: you gotta do the hard work so our users can enjoy reading it.

Reply:
also please put at least TWO seachat_ref in the table